pkg/kubectl/scale.go

```diff
@@ -180,11 +197,63 @@ func (scaler *ReplicationControllerScaler) Scale(namespace, name string, newSize
 	return nil
 }
 
-// ScalerClient abstracts access to ReplicationControllers.
+// ScaleSimple is responsible for updating job's parallelism.
+func (scaler *JobScaler) ScaleSimple(namespace, name string, preconditions *ScalePrecondition, newSize uint) (string, error) {
```

Comment:
Hmm... looking at this code I don't see any real usage of that returned value. In that case I'll remove it, the method will be returning just error, unless somebody else will object.